### ISSUE TITLE
INTERFACING LTE MODULE WITH LPC 1769

### ISSUE THREAD

**Issue by mikeshams** [ankitpatel9300/Capstone-Project--Summer-2020#5]

**Comment by ankitpatel9300** [comment]
> LTE module gets restarted automatically frequently. 
> Finding the RCA for issue. Issue was not seen before.
> Confirmed that restart is not triggered by Software

**Comment by ankitpatel9300** [comment]
The last couple of days Issue was not seen.
Search on the internet and it looks like static charge was the culprit.

**Comment by mikeshams** [comment]
1. What issues did you or others might face with connection of the LTE Module to an LPC 1769 in terms of Hardware and Software?

**Comment by ankitpatel9300** [comment]
So, Below issues can arise with LTE module interfacing.

1. Can not communicate through the UART port.
    -check jumpers on Module >> Chack driver code of UART >> Confirm logic level at both ends.
    
2. LTE module does not wake up.
    check power on the module.

3 LTE module does not connect to the network.
  Check SIM status >> confirm freq Band with Operator and Module >> check internet connection by inserting SIM in mobile.

4 LPC is not able to read proper data from Module
 - Check baud rate > start bit >> stop bit and parity bits.
 - Also, disable Auto-baud rate >> Check default baud rate and match with LPC UART baud rate.

Another issue I faced os of Frequent restart of the module.
I commented on that already.
